search-commons/src/main/java/no/unit/nva/search2/resource/Constants.java

```diff
@@ -176,6 +176,7 @@ public final class Constants {
 
     public static final List<AbstractAggregationBuilder<? extends AbstractAggregationBuilder<?>>>
         RESOURCES_AGGREGATIONS = List.of(
+        licenseHierarchy(),
```

Comment:
when you splitt into seperate hierachies, speed of query is reduced, and it breaks the pattern.